sphinx_rtd_theme/layout.html

```diff
@@ -162,16 +163,28 @@
                 target="_blank"
               >
                 <span class="gh-ico" aria-hidden="true"></span>
-                <span class="gh-text">Star</span>
+                <span class="gh-text">GitHub Stars</span>
               </a>
               <a
                 class="gh-count"
                 href="https://github.com/cilium/cilium/stargazers"
                 rel="noopener noreferrer"
                 target="_blank"
                 aria-label={`${this.state.stars} stargazers on GitHub`}
+                style="margin-left: 0px;"
               >
               </a>
+
+              <a
+                class="gh-btn"
+                href="https://cilium.herokuapp.com/"
+                rel="noopener noreferrer"
+                target="_blank"
+                style="margin-left: 15px;"
+              >
+                <img style="height: 1em;" src="https://raw.githubusercontent.com/cilium/cilium.io/main/src/icons/slack.inline.svg" alt="slack_icon">
```

Comment:
Hey @qmonnet, I added the SVGs locally, but there rose a new issue. 

I couldn’t get it to render even when I am accessing the right file.
<img width="575" alt="Screenshot 2022-07-03 at 17 27 59" src="https://user-images.githubusercontent.com/42554056/177048644-b6cfebb4-42be-45b9-8dee-8d0988c2b149.png">

I spent hours trying to fix it using different HTML tags as it seems to be a known issue with SVGs but to no avail.

I then thought of a way around it:

Since the goal is for it not to depend on the cilium.io repository, I put the src for each svg to the link it will be hosted on the theme repo. Similar to this -> `https://raw.githubusercontent.com/Kikiodazie/cilium_sphinx_rtd_theme/pr/add-svg-locally/docs/_static/icons/slack.inline.svg`

I tested it with my fork and it works just fine. What do you think about it? Though it works, if you can help me figure out getting the SVG to render without the hassle that would be perfect. 

I hope this meets you well.

Reply:
Of the two solutions, I would much rather have inline SVGs. Using the full URL to the GH repo is so likely to break and be hard to debug if we ever rename a branch or a repo at some point.

But if you didn't manage to link the files, this is maybe simply because they were not copied from the theme repo to the `_static` directory for the docs along with the CSS. Have you tried adding your `icons` directory to [`package_data`](https://github.com/Kikiodazie/cilium_sphinx_rtd_theme/blob/v1.0/setup.py#L117)?

Reply:
So i tried adding the `icons` directory to the [package_data](https://github.com/Kikiodazie/cilium_sphinx_rtd_theme/blob/v1.0/setup.py#L117), and it didn't work. 

So I just added the inline SVGs. Do check it out and let me know if it works.

Reply:
Thanks, will have a look next week :+1:

Reply:
No problem. Thank you. Do have a great weekend.

Reply:
So adding the SVGs locally works just fine for me. Here's what I had to do:

- I reused your PR but discarded the last commit (so I'm in the state where you added the icons locally but it didn't work, and before you added them inline in the HTML).
- I moved the SVGs from `docs/_static/icons` to `sphinx_rtd_theme/static/icons`. This is where they should be. I probably got you confused when mentioning `Documentation/_static` in my comment below, but that's where I wanted to see them end up when building the docs in the Cilium repo, not in _this_ repo. Apologies for not being more explicit.
- I added `static/icons/*.svg` to `package_data` in `setup.py`
- I fixed the paths in `sphinx_rtd_theme/layout.html`, respecting the templating syntax, to make it point to the correct location.

This is available on [this branch](https://github.com/qmonnet/sphinx_rtd_theme/commits/test_svgs), see the last commit.

Reply:
Thank you very much for this walk-through, Quentin, I sincerely appreciate it. 

Thank you for the branch I just went ahead and rebased it on this my branch 😁🫣